modules/container/CMakeSources.cmake

```diff
@@ -0,0 +1,17 @@
+set(MODULE_PATH
+        ${CMAKE_CURRENT_SOURCE_DIR}/claws/container)
+
+set(MODULE_PUBLIC_HEADERS
+        "${MODULE_PATH}/array_ops.hpp"
```

Comment:
I don't think this belongs here. But it's deprecated anyway, we'll remove this when `claws::vect` is in a good state.